ruby/pom.xml

```diff
@@ -78,7 +78,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
-            <version>3.0.0-pre</version>
+            <version>3.0.0-alpha-3-pre</version>
```

Comment:
We have a list of places to update the version in our internal step-by-step release checklist. Agreed it would be nice to automate but right now seems like it might be more scripting/file autogeneration mess than it's worth...